app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -106,21 +106,24 @@ public void onCreate(Bundle savedInstanceState) {
         RecentTasksStyler.styleRecentTasksEntry(this);
 
         if (!Utils.isEmulator()) {
-
-            mGoogleApiClient = new GoogleApiClient.Builder(this)
-                    .addApi(Plus.API)
-                    .addApi(Games.API)
-                    .addApi(AppStateManager.API)
-                    .addScope(Plus.SCOPE_PLUS_LOGIN)
-                    .addScope(Plus.SCOPE_PLUS_PROFILE)
-                    .addScope(Games.SCOPE_GAMES)
-                    .build();
+            createGoogleApiClient();
         }
 
         mAchievementActionHandler =
                 new AchievementActionHandler(getHandler(), mGoogleApiClient, this);
     }
 
+    protected void createGoogleApiClient() {
+        mGoogleApiClient = new GoogleApiClient.Builder(this)
+                .addApi(Plus.API)
+                .addApi(Games.API)
+                .addApi(AppStateManager.API)
```

Comment:
I created new `protected void createGoogleApiClient()` in GdgActivity and override this in `DeepLinkActivity` and add `.addApi(AppInvite.API)` there. We don't need it in every page.